release-notes/General/General_Feature_Release_10.5/General_Feature_Release_10.5.1.md

```diff
@@ -99,6 +99,95 @@ From now on, users will be allowed to give feedback (positive or negative) on al
+
+> [!IMPORTANT]
+> Once the element configuration has been moved from disk to database, there is no good way to revert this change, which means that if you were to disable Swarming again, you would lose all your elements, leaving your DMS with a lot of lingering references to non-existing elements. For instructions on how to disable Swarming and **partially** recover your elements, see [Partially rolling back Swarming](xref:SwarmingRollback).
+
+##### Capabilities
+
+Currently, the Swarming feature provides the following capabilities:
+
+- As a DataMiner System Admin, you can apply maintenance (e.g. Windows updates) on a live cluster, Agent by Agent, by temporarily moving functionalities away to other Agents in the cluster.
+
+- As a DataMiner System Admin, you can easily extend your system with an extra node and move functionalities from existing nodes to new nodes, so you can rebalance your cluster.
+
+- Swarming makes it possible to recover functionalities from failing nodes by moving activities hosted on such a node to the remaining nodes.
+
+In a later iteration, the Swarming feature will also be able to assist in making rolling DataMiner software updates on live clusters possible, with limited downtime of specific functionality.
+
+> [!NOTE]
+> The above capabilities are possible with limited downtime and as long as there is spare capacity.
+
+##### Limitations
+
+Some functionality is currently not yet supported with the Swarming feature. These are the most important limitations at the moment:
+
+- Swarming is not available in DataMiner Systems with a [storage per DMA setup](xref:Configuring_storage_per_DMA) (Cassandra or MySQL).
+
+- Swarming is not available in DataMiner Systems with [Failover](xref:About_DMA_Failover).
+
+- Swarming is not available in DataMiner Systems where the [*LegacyReportsAndDashboards* soft-launch option](xref:Overview_of_Soft_Launch_Options#legacyreportsanddashboards) is enabled.
+
+- Swarming is currently limited to [basic elements](xref:SwarmingElements).
+
+> [!NOTE]
+> [Prerequisite checks](xref:EnableSwarming#running-a-prerequisites-check) are in place to prevent the enabling of the Swarming feature when non-supported objects are present. Where possible, you will also be prevented from configuring or creating these on a Swarming-enabled system.
+
+##### Required user permissions
+
+To enable the Swarming feature, the [Admin Tools](xref:DataMiner_user_permissions#modules--system-configuration--tools--admin-tools) user permission is required.
+
+Once the feature has been enabled, users will need the [Swarming](xref:DataMiner_user_permissions#modules--swarming) user permission to trigger any swarming actions. Users that previously had the [Import DELT](xref:DataMiner_user_permissions#general--elements--import-delt) and [Export DELT](xref:DataMiner_user_permissions#general--elements--import-delt) user permissions before Swarming was enabled will automatically also get the *Swarming* user permission.
```

Comment:
note: automatically assigning the Swarming permission is something done only once, when upgrading from a version that does not have the flag present to one that does have it